src/api/custom-resolvers/mutations/submit-application/index.ts

```diff
@@ -27,21 +27,24 @@ const submitApplication = async (root: any, variables: SubmitApplicationVariable
     })) as Application;
 
     if (application) {
-      const isInProgress = application.status === APPLICATION.STATUS.IN_PROGRESS;
+      const { status, submissionDeadline, submissionCount } = application;
+
+      const isInProgress = status === APPLICATION.STATUS.IN_PROGRESS;
 
       const now = new Date();
 
       // check the current date vs submission deadline
-      const validSubmissionDate = isAfter(new Date(application.submissionDeadline), now);
+      const validSubmissionDate = isAfter(new Date(submissionDeadline), now);
 
-      const canSubmit = isInProgress && validSubmissionDate;
+      const canSubmit = isInProgress && validSubmissionDate && submissionCount === 0;
```

Comment:
Might be ideal to create a variable as below

```javascript
const firstSubmission = submissionCount === 0;
const canSubmit = isInProgress && validSubmissionDate && firstSubmission;
```

Reply:
Good shout, have changed and using `isFirstSubmission` variable.